src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -3280,6 +3280,8 @@ gregtech.machine.multiblock.universal.controller_information=Controller block fo
 
 # Multiblock machine parts
 gregtech.machine.item_bus.import.tooltip=Item Input for Multiblocks
+gregtech.machine.disallow.hatch.tooltip=Sharing between Multiblocks: §4Disabled
+gregtech.machine.allow.bus.tooltip=Sharing between Multiblocks: §aEnabled
```

Comment:
We may already have these translations elsewhere in the lang file, if not these should be converted to keys like `gregtech.universal.disabled` and enabled to be reused when needed